model_analyzer/config/input/config_command_profile.py

```diff
@@ -660,6 +678,15 @@ def _preprocess_and_verify_arguments(self):
                 raise TritonModelAnalyzerException(
                     "triton_docker_image provided but is empty.")
 
+        if self.triton_launch_mode == 'C_API':
+            if self.triton_server_flags or self.triton_server_environment:
```

Comment:
I understand why it cannot set `triton_server_flags` but why can't we set the `triton_server_environment`?

Reply:
Ah I suppose we can set the flags during the perf_analyzer launch.